recipes/pylibseq/meta.yaml

```diff
@@ -1,29 +1,31 @@
 package:
   name: pylibseq
-  version: "0.2.0"
+  version: "0.2.1"
 
 source:
-  url: https://files.pythonhosted.org/packages/ac/22/06172cf9ff7e62a9c83cd8574c3994acaf0d3e16d9472f236e4a71a9692b/pylibseq-0.2.0.tar.gz
-  sha256: 04c9ba91e79876c6c31229eeda70e49600afb81fa78270ae4f6f58b847b8b72f
+  url: https://files.pythonhosted.org/packages/86/f9/0da11046680d74ceba0aade1bca859951479eacc4bda2bf062a568216cb4/pylibseq-0.2.1.tar.gz 
+  sha256: 6bfc509f90832f38e9c9ca862b8f57e35a68ba2b218a1092e3c18affef41f6dd
 
 build:
-  skip: True # [py27]
+  skip: True # [py27] 
+  skip: True # [osx]
   # If this is a new build for the same version, increment the build
   # number. If you do not include this key, it defaults to 0.
-  number: 2
+  number: 1
```

Comment:
this should be 0

Reply:
Thanks. I'll change the build number.

svgwrite is a dependency of msprime, which is a listed dependency of my project.

Reply:
@bgruening This is the error being reported:

```
RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.org/simple/svgwrite/
```